aggregators/internal/hdrhistogram/hdrhistogram_test.go

```diff
@@ -21,10 +21,11 @@ func TestMerge(t *testing.T) {
 
 	for i := 0; i < 1_000_000; i++ {
 		v1, v2 := rand.Int63n(3_600_000_000), rand.Int63n(3_600_000_000)
-		hist1.RecordValues(v1, 11)
-		histRep1.RecordValues(v1, 11)
-		hist2.RecordValues(v2, 111)
-		histRep2.RecordValues(v2, 111)
+		c1, c2 := rand.Int63n(1_000), rand.Int63n(1_000)
```

Comment:
The rand here makes sense to not have to check for every value.
But since we have an explicit condition for `0`, should we also be checking explicitly with that value every time?

Reply:
Sounds good. I have added it :D